### PR TITLE
added docs for upgrading 0.2.1 and 0.3-pre versions

### PR DESCRIPTION
**Pull request by dolanjs**
Added two docs to cover upgrading from a 0.2.1 version to 0.3.3 and different document for upgrading 0.3-pre to 0.3.3 version.

In the hackpad these were 1 document. @conorsch thought it would be easier for the organizations if the documents were split up.

### CODE REVIEW

File: docs/0.2.1-upgrade-to-0.3.3.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## Prerequisites
+
+* A networked Admin tails workstation configured with the SecureDrop repo cloned to the `Persistent` directory.
+
+* Physical/remote access to the app and monitor servers
+
+## Upgrade 0.2.1 steps
+
+There is a `0.2.1_collect.py` python script that will backup a SecureDrop version 0.2.1 app server. This script will need to be ran on the app server prior to performing a clean install of SecureDrop 0.3.x. The direction below assume that the admin has physical access to the app server.
```
Comment (conorsch) [inline docs/0.2.1-upgrade-to-0.3.3.md]: Nit: "need to be run"

**Comment by conorsch** [comment]
lgtm. Later on we can add a generic `docs/upgrade.md` to manage the different migration paths if necessary, but these targeted docs should get folks up to speed with 0.3.3.

File: docs/0.3-pre-upgrade-to-0.3.3.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+`Error: the app or mon ansible_ssh_host in Ansible's inventory file is not an .onion address.`
+
+* Our new provisioner, Ansible, must be run over Tor in order to reach the servers. Replace the IP addresses in `/install_files/ansible-base/inventory` with the .onion hostnames for the App and Mon server's Tor hidden services for SSH.
+
+`Error: can't connect to the Application or Monitor Server via SSH.`
+
+* Something's wrong and we can't connect. You can re-run the script to try again.
+
+** Is the Vidalia connection indicator green?
```
Comment (garrettr) [inline docs/0.3-pre-upgrade-to-0.3.3.md]: This is not the correct syntax for nested bulleted lists in Markdown (it is correct for Creole).

File: docs/0.2.1-upgrade-to-0.3.3.md (hunk 1 above)
Comment (garrettr) [inline docs/0.2.1-upgrade-to-0.3.3.md]: Nit: "directions below"

File: docs/0.2.1-upgrade-to-0.3.3.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+`git clone https://github.com/freedomofpress/securedrop`
+
+* Change directory into the securedrop folder
+
+`cd securedrop`
+
+* Checkout and verify the current release
+
+`git checkout tags/0.3.3`
```
Comment (garrettr) [inline docs/0.2.1-upgrade-to-0.3.3.md]: Hm, are you sure you based this on the latest version of the [Hackpad](https://securedrop.hackpad.com/Upgrade-to-0.3.3-jis3fmQ3AnE)? I made some edits that I am not seeing in this document. For example, `tags/` is unnecessary, so I changed this line to `git checkout 0.3.3` in the Hackpad.

**Comment by garrettr** [comment]
Did you hand-convert this from Hackpad to Markdown? I use [this Chrome extension](https://chrome.google.com/webstore/detail/hackpad-downloader/jhkfiecgggifjcoliemlcajokapmcpih?hl=en) to export Hackpads to Markdown.

File: docs/0.2.1-upgrade-to-0.3.3.md (hunk 3 above)
Comment (dolanjs) [inline docs/0.2.1-upgrade-to-0.3.3.md]: I did use the current version from hackpad. You only made that change in one place, not all of the places it was used.

**Pull request merged**